### PR TITLE
OCPBUGS-17654: cm: reorder setting of sched_load_balance for sandbox slice

### PR DESCRIPTION
**Pull request by haircommander**
#### What type of PR is this?

/kind bug 
#### What this PR does / why we need it:
If we call mgr.Apply() first, libcontainer's cpusetCopyIfNeeded() will copy the parent cpuset and set load balancing to 1 by default. This causes the kernel to set the cpus to not load balanced for a brief moment which causes churn.

instead, create the cgroup and set load balance, then have Apply() copy the values into it.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
fix a bug in cpuset load balancing where cpusets flip between load balanced and not due to an ordering issue.
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/kubelet/cm/cgroup_manager_linux.go [hunk 1]
```diff
@@ -464,6 +464,25 @@ func (m *cgroupManagerImpl) Create(cgroupConfig *CgroupConfig) error {
...
+	// disabled, but the slices can contain all cpus (as the guaranteed cpus are known dynamically).
+	// Note: this should be done before Apply(-1) below, as Apply contains cpusetCopyIfNeeded(), which will
+	// populate the cpuset with the parent's cpuset. However, it will be initialized to sched_load_balance=1
+	// which will cause the kernel to move all cpusets out of their isolated sched_domain, causing unnecessary churn.
+	if m.cpuLoadBalanceDisable && !libcontainercgroups.IsCgroup2UnifiedMode() {
+		path := manager.Path("cpuset")
+		if path == "" {
+			return fmt.Errorf("Failed to find cpuset for newly created cgroup")
+		}
+		if err := os.Mkdir(path, 0o755); err != nil {
```
Comment (rphillips) [inline pkg/kubelet/cm/cgroup_manager_linux.go:478]: Sorta think we should catch the exists case here too

File: pkg/kubelet/cm/cgroup_manager_linux.go [hunk 2]
```diff
@@ -464,6 +464,25 @@ func (m *cgroupManagerImpl) Create(cgroupConfig *CgroupConfig) error {
...
+			return fmt.Errorf("Failed to find cpuset for newly created cgroup")
+		}
+		if err := os.Mkdir(path, 0o755); err != nil && !os.IsNotExist(err) {
+			return fmt.Errorf("failed to create cpuset for newly created cgroup: %w", err)
+		}
+		if err := cgroups.WriteFile(path, "cpuset.sched_load_balance", "0"); err != nil {
+			return err
+		}
+	}
+
```
[the remark this answers is not in the document]
Comment (haircommander) [reply pkg/kubelet/cm/cgroup_manager_linux.go:485]: https://github.com/openshift/kubernetes/pull/1665/files#diff-2f332b11d91d53583a6eeca75fc6e124aee6bc547a264e6987b89b1c786e0674R473 it is!

[the remark this answers is not in the document]
Comment (kolyshkin) [reply pkg/kubelet/cm/cgroup_manager_linux.go:485]: Ah, missed it, thanks! LGTM

**Review by kolyshkin — APPROVED** [review]
lgtm

**Pull request merged**